packages/devtools_app/lib/src/screens/performance/enhance_tracing.dart

```diff
@@ -0,0 +1,379 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import '../../primitives/auto_dispose_mixin.dart';
+import '../../service/service_extension_manager.dart';
+import '../../service/service_extensions.dart' as extensions;
+import '../../shared/common_widgets.dart';
+import '../../shared/globals.dart';
+import '../../shared/theme.dart';
+import '../../ui/service_extension_widgets.dart';
+import 'performance_screen.dart';
+
+class EnhanceTracingButton extends StatelessWidget {
```

Comment:
this class was moved over from performance_screen.dart with a couple changes (marked below)